### ISSUE TITLE
Aviator Future Dead?

### ISSUE THREAD

**Issue by Aldous-Huxley** [WhiteHatSecurity/Aviator#78]
I need no responses from a community that appears to be dead.

**Comment by alexberry721** [comment]
@Aldous-Huxley isn't that a little like the pot calling the kettle black?

**Comment by Darque** [comment]
Sadly, I have to agree. It would be nice to get a response from someone close to the project so we can understand what happened. A browser that is not being updated is dead indeed. Such a pity as it was a great idea.

**Comment by Andreas-Schoenefeldt** [comment]
What a pitty... it needs maybe 2 minutes to come to the same conclusion. Fare well Aviator.

**Comment by captvic** [comment]
I did not know they stopped updating until today, when the default home page came up with page not found. Started to investigate, nothing on their site until I did search for Aviator and then landed here. I appreciate all their hard work on this product but it would have been nice if they had changed the home page a while back to let users know what was going on. I thought that this product was being maintained by a security company and now am concerned when did they stop patching and was I exposed to any vulnerabilities. Not to be ungrateful, but just let me know when you decide to stop working on a product. Adios.

**Comment by DinisCruz** [comment]
I agree it was a good idea, just not implemented the right way and not enough community support. At least the code is open source so it might live another day
